t/ssl.t

```diff
@@ -2168,3 +2170,84 @@ client ip: 127.0.0.1
 [error]
 [alert]
 [emerg]
+
+
+
+=== TEST 21: yield during doing handshake with client which uses low version OpenSSL
```

Comment:
Adding `--- ONLY` to this test case and running this test file yields:

```
/bin/openssl
OpenSSL 1.0.2m-fips  2 Nov 2017
t/ssl.t .. # I found ONLY: maybe you're debugging?
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] still could not bind()
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] still could not bind()
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:4443 failed (98: Address already in use)
```

Running other test blocks individually in this test file does not have any problems.